starknet_devnet/account.py

```diff
@@ -47,6 +47,15 @@ def get_contract_class(cls):
             cls.CONTRACT_CLASS = ContractClass.load(load_nearby_contract(cls.CONTRACT_PATH))
         return cls.CONTRACT_CLASS
 
+    def to_json(self):
+        """Return json account"""
+        return {
+            "initial_balance": self.initial_balance,
```

Comment:
Would it be better to return actual balance? It shouldn't take much longer to retrieve that, right?

Reply:
I just return here class properties stat is available without additional queries as part of a class in relation to the initial seed. The actual balance is out of that scope as it isn't propped but needs interaction with the chain?

Reply:
FeeToken has a `get_balance` method that could be used. But for now we can keep it as it is, and in a next PR we could add a `balance` property (so there would be both initial_balance and balance)